res/values/arrays.xml

```diff
@@ -2,74 +2,74 @@
 <resources>
   <string-array name="pref_layout_values">
     <item>none</item>
-    <item>azerty</item>
-    <item>qwerty</item>
-    <item>qwerty_pt</item>
-    <item>qwerty_es</item>
-    <item>qwerty_el</item>
-    <item>qwerty_ko</item>
-    <item>qwerty_lv</item>
-    <item>qwerty_no</item>
-    <item>qwerty_sv_se</item>
-    <item>qwerty_hu</item>
-    <item>qwerty_tr</item>
-    <item>qwerty_pl</item>
-    <item>qwerty_vi</item>
-    <item>ru_jcuken</item>
-    <item>qwertz</item>
-    <item>qwertz_cs</item>
-    <item>qwertz_de</item>
-    <item>qwertz_hu</item>
-    <item>qwertz_sk</item>
-    <item>bgph1</item>
-    <item>dvorak</item>
-    <item>colemak</item>
-    <item>neo2</item>
-    <item>bone</item>
-    <item>jcuken_ua</item>
-    <item>bangla</item>
-    <item>devanagari_1</item>
-    <item>devanagari_2</item>
-    <item>he_il_1452_1</item>
-    <item>he_il_1452_2</item>
-    <item>ar_pc</item>
-    <item>ar_alt</item>
+    <item>arab_ara_ibm</item>
```

Comment:
The app being designed for developers, I'd prefer to see programming layouts at the beginning of the list: QWERTY, Colemak, Dvorak.

Reply:
It's about sorting with which order. Currently it is a mixture: ISO 15924 first, name second. It needs to be changed for the complexity.

Maybe https://github.com/Julow/Unexpected-Keyboard/issues/325 will help: in addition to writing systems, there is a “[programming]” group including them.

Reply:
https://github.com/Julow/Unexpected-Keyboard/issues/325 is a good solution but the groups still have to be sorted anyway.

Reply:
Yes, it is. _programming_ will be at the top of the first menu together with writing systems, languages will be on the second menu and layouts for that language will be on the third. Then all layouts can be sorted by their codes.

Reply:
Thanks. In the meantime, you can separate groups of items by adding empty lines. That way, items inside a group can be sorted lexicographically while groups are ordered as we want.

I understand that the list is hard to re-order confidently. I plan to fix https://github.com/Julow/Unexpected-Keyboard/issues/329 at some point to solve the issue of the two arrays going out of sync.

Reply:
If you have the time to implement it, that would be wonderful! Please keep this in a separate PR.